### PR TITLE
Added group-update functionality.

### PR DESCRIPTION
**Pull request by carlism**
### Description of the Change

Added the group-update command as per the request to accept a json representing the entire group.
Tested this by requesting a previously created group's json and changed the email address and submitted it back to group-update.

### Why Should This Be In The Package?

It adds the group-update function addressing an open issue.

### Benefits

Groups may be updated using the cli

### Possible Drawbacks

Not exactly fine grained access.

### Verification Process

A successful response was received from the API and I confirmed that the data changed.

### Applicable Issues (Optional)

Issue #45

### CODE REVIEW

**Comment by pauljamescleary** [comment]
My only question is that, is there help text for the cli that needs to be updated with these new commands @mdb @carlism ?

**Comment by carlism** [comment]
@pauljamescleary Absolutely right!  README is now updated.

File: tests/vinyldns_tests.bats [hunk 1]
```diff
@@ -49,6 +49,18 @@ load test_helper
...
+  ok_group=$($ew --op json group --name "ok-group")
+  updated_group="$(echo ${ok_group} | sed 's/test@test.com/update@update.com/g')"
+  run $ew group-update \
+    --json "${updated_group}"
+
+  [ "${output}" = "${fixture}" ]
+}
+
+
+
```
Comment (pauljamescleary) [inline tests/vinyldns_tests.bats:63]: my only nitpic is too many new lines here (is there an autoformatter that can be used I wonder?)

Comment (carlism) [reply tests/vinyldns_tests.bats]: I adjusted the formatting.  No formatter readily available for bats files in VSC.

**Review by mdb — APPROVED** [review]
LGTM @carlism ! Thank you.

**Pull request merged**